Source/ASNetworkImageNode.mm

```diff
@@ -42,41 +42,39 @@ @interface ASNetworkImageNode ()
   // The download identifier that we have set a progress block on, if any.
   id _downloadIdentifierForProgressBlock;
 
-  BOOL _imageLoaded;
-  BOOL _imageWasSetExternally;
   CGFloat _currentImageQuality;
   CGFloat _renderedImageQuality;
-  BOOL _shouldRenderProgressImages;
 
-  struct {
-    unsigned int delegateWillStartDisplayAsynchronously:1;
-    unsigned int delegateWillLoadImageFromCache:1;
-    unsigned int delegateWillLoadImageFromNetwork:1;
-    unsigned int delegateDidStartFetchingData:1;
-    unsigned int delegateDidFailWithError:1;
-    unsigned int delegateDidFinishDecoding:1;
-    unsigned int delegateDidLoadImage:1;
-    unsigned int delegateDidLoadImageFromCache:1;
-    unsigned int delegateDidLoadImageWithInfo:1;
-  } _delegateFlags;
-
-  
-  // Immutable and set on init only. We don't need to lock in this case.
+    // Immutable and set on init only. We don't need to lock in this case.
   __weak id<ASImageDownloaderProtocol> _downloader;
-  struct {
-    unsigned int downloaderImplementsSetProgress:1;
-    unsigned int downloaderImplementsSetPriority:1;
-    unsigned int downloaderImplementsAnimatedImage:1;
-    unsigned int downloaderImplementsCancelWithResume:1;
-    unsigned int downloaderImplementsDownloadWithPriority:1;
-  } _downloaderFlags;
 
   // Immutable and set on init only. We don't need to lock in this case.
   __weak id<ASImageCacheProtocol> _cache;
   struct {
-    unsigned int cacheSupportsClearing:1;
-    unsigned int cacheSupportsSynchronousFetch:1;
-  } _cacheFlags;
+      unsigned int delegateWillStartDisplayAsynchronously:1;
```

Comment:
Can you add a comment above to indicate that keeping all of these flags in one struct reduces the nodes memory footprint?